packages/components/src/FormField/FormField.tsx

```diff
@@ -173,7 +173,7 @@ export function FormField(props: FormFieldProps) {
 
         function handleClear() {
           handleBlur();
-          setValue(controlledName, undefined, { shouldValidate: true });
```

Comment:
<img width="988" alt="image" src="https://github.com/GetJobber/atlantis/assets/11843215/09c22f80-260f-4fe9-ad49-71c857323247">
https://react-hook-form.com/docs/useform/setvalue


while that doesn't directly recommend you to use an empty string, there's some related functionality around the default value that says to use an empty string or null

<img width="940" alt="image" src="https://github.com/GetJobber/atlantis/assets/11843215/1b5e0314-c213-480c-b128-7cf7bed54a92">


null does not work, so went with an empty string